src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CaseFinder.java

```diff
@@ -114,6 +114,10 @@ private void logCaseRetrievalResultBasedOnLegacyIdSearch(
         try {
             return Optional.of(ccdApi.getCase(ccdCaseRef, jurisdiction));
         } catch (CaseNotFoundException e) {
+            log.info("Case wasn't found by CCD ID {}", ccdCaseRef);
+            return Optional.empty();
+        } catch (InvalidCaseIdException e) {
```

Comment:
Can we please add a test case for this ?

Reply:
done